velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -62,7 +62,11 @@ int main(int argc, char** argv) {
       "chr",
       "replace",
       "might_contain",
-      "unix_timestamp"};
+      "unix_timestamp",
```

Comment:
In general for better relibility I always recommend using the simple function interface, from my experiance workin in fuzzer its almost impossible not to get bugs in complicated vector functions/

The shall be used as final resource when performance gap is really big and important.